engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.module.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
In the end I decided to keep it under capsules-content/ as moving it caused me some additional troubles with loading the modules. I might move it if there's some time left.